tests/all/js_globals/Reflect.rs

```diff
@@ -0,0 +1,703 @@
+#![allow(non_snake_case)]
+
+use project;
+
+#[test]
+fn apply() {
+    project()
+        .file(
+            "src/lib.rs",
+            r#"
+            #![feature(proc_macro, wasm_custom_section)]
+
+            extern crate wasm_bindgen;
+            use wasm_bindgen::prelude::*;
+            use wasm_bindgen::js;
+
+            #[wasm_bindgen]
+            pub fn apply(target: &js::Function, this_argument: &JsValue, arguments_list: &js::Array) -> JsValue {
+                let result = js::Reflect::apply(target, this_argument, arguments_list);
+                let result = match result {
+                    Ok(val) => val,
+                    Err(_err) => "TypeError".into()
```

Comment:
Have you looked up the contents of `_err`? Maybe it could be used as a return value here (I'm not sure what's there, but it could be worth checking).

Reply:
_err indeed contains the actual error message. Again, we should probably apply those changes [here](https://github.com/rustwasm/wasm-bindgen/blob/2694dd4dee28e36981e026e60d5bdf141f51562d/tests/all/js_globals/Number.rs#L75) as well.